requirements.txt

```diff
@@ -2,7 +2,7 @@ jsonschema>=2.6
 networkx>=2.2;python_version>'3.5'
 networkx>=2.2,<2.4;python_version=='3.5'
 retworkx>=0.3.2
-numpy>=1.17
+numpy>=1.17,<=1.18
```

Comment:
This won't have the desired effect in all locations, because the source of truth for this list is the requirements list in the setup.py. That being said, If we're prepping this to merge, I would probably just pin this in the constraints file (that's what it's there for). Then when we have a fix (or a confirmed issue on numpy submitted) we can either add `!=1.19.0` to the requirements list in the setup.py and remove the constraint line, or just remove the constraint (if it's a terra side change).

Reply:
oh.. I forgot the `setup.py`! Thanks!

Reply:
Moved to `constraints.txt` in  220e9b3c3704578f068ae9795abbf621e5f1da7b